### PR TITLE
Auth user

### PR DESCRIPTION
**Pull request by IrynaSpyrydonova**
Guys, here I removed files from starter that caused a lot of problems, should change all package.json, because mostly it was problems with that. I checked everything even in heroku, everything works. I know that Pavel changed a bit structure of files in his PR,I lam absolutely ok his suggestions but already after this PR with all packages and bugs fixed. Pavel, my apologies  for that, i ready to fix all conflicts and put all your changes by myself.

### CODE REVIEW

**Comment by IrynaSpyrydonova** [comment]
Guys, the conflict that we have now i will fix, but only after your approval.

**Review by pavelbidenko2018 — APPROVED** [review]

**Comment by pavelbidenko2018** [comment]
> Guys, the conflict that we have now i will fix, but only after your approval.

Approved 👍

**Pull request merged**
